src/experimental/build/index.ts

```diff
@@ -139,37 +116,39 @@ async function handleNewCommand(): Promise<void> {
     );
 
     await runStableNewCommand(azleVersion, templatePath, httpServer === true);
+    await installAzleExperimentalDepsForNewProject(projectName);
+}
+
+async function installAzleExperimentalDepsForNewProject(
+    projectName: string
+): Promise<void> {
+    const packageJsonPath = join(projectName, 'package.json');
+    const currentPackageJsonContents = await readFile(packageJsonPath, {
+        encoding: 'utf-8'
+    });
+    let packageJson = JSON.parse(currentPackageJsonContents);
+    packageJson.devDependencies['azle-experimental-deps'] =
```

Comment:
`packageJson.dependencies` instead of `devDependencies`

Reply:
This should be done everywhere I think

Reply:
done